src/user.rs

```diff
@@ -295,3 +312,14 @@ async fn pick_random_room(rooms: &RwLock<Vec<OwnedRoomId>>) -> Option<OwnedRoomI
         .choose(&mut rand::thread_rng())
         .map(|room| room.to_owned())
 }
+
+/// Get random value for ticks to live related to the total of ticks in simulation,
+/// so users can be short or long lived.
+fn get_ticks_to_live(config: &Config) -> usize {
+    let mut rng = rand::thread_rng();
+    let short_lived = rng.gen_bool(1. / 2.);
```

Comment:
Wondering if this should be the same as:
```suggestion
    let short_lived = rng.gen_bool(0.5);
```
Either way, could we make this configurable by the user to try out different scenarios?

Reply:
yes, sure!